shared/common-adapters/radio-button.desktop.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import * as React from 'react'
+import Text from './text'
+import glamorous from 'glamorous'
+import type {Props} from './radio-button'
+import {globalStyles, globalColors, transition} from '../styles'
+
+export const RADIOBUTTON_SIZE = 14
+export const RADIOBUTTON_MARGIN = 8
+
+const RadioButton = ({disabled, label, onSelect, selected, style}: Props) => {
+  const clickableStyle = disabled ? {} : globalStyles.clickable
+
+  const StyledDiv = glamorous(glamorous.Div)(({border, disabled, selected}) => ({
```

Comment:
This should be pulled outside of RadioButton so its not remade everytime this renders